src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -252,26 +254,27 @@ private InitCodeView buildInitCodeView(
       MethodTransformerContext context, Iterable<InitCodeNode> SpecItemNodes) {
     List<InitCodeLineView> surfaceLines = new ArrayList<>();
     for (InitCodeNode item : SpecItemNodes) {
-      switch (item.getLineType()) {
-        case StructureInitLine:
-          surfaceLines.add(generateStructureInitCodeLine(context, item));
-          continue;
-        case ListInitLine:
-          surfaceLines.add(generateListInitCodeLine(context, item));
-          continue;
-        case SimpleInitLine:
-          surfaceLines.add(generateSimpleInitCodeLine(context, item));
-          continue;
-        case MapInitLine:
-          surfaceLines.add(generateMapInitCodeLine(context, item));
-          continue;
-        default:
-          throw new RuntimeException("unhandled line type: " + item.getLineType());
-      }
+      surfaceLines.add(generateSurfaceInitCodeLine(context, item));
     }
     return surfaceLines;
   }
 
+  private InitCodeLineView generateSurfaceInitCodeLine(
+      MethodTransformerContext context, InitCodeNode SpecItemNode) {
```

Comment:
the variable SpecItemNode should be lower camel; it looks like there was a previous mistake that put SpecItemNodes in upper camel - it should be lower camel too.